### PR TITLE
Install plugins

### PR DESCRIPTION
**Pull request by djaglowski**
## **Please check that the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Add a changelog entry (for non-trivial bug fixes / features)
- [ ] CI passes

### CODE REVIEW

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Average (%) | Memory Average (MB) |
| --- | --- | --- | --- |
|1|1000|2.3448615|28.550781|
|1|5000|6.1898108|31.737743|
|1|10000|9.224435|44.775322|
|1|50000|36.88012|246.70218|
|1|100000|63.60469|220.46808|
|10|100|4.017329|25.715248|
|10|500|8.396685|34.407597|
|10|1000|11.345142|43.23882|
|10|5000|41.98305|446.93332|
|10|10000|71.276405|318.58136|

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Average (%) | Memory Average (MB) |
| --- | --- | --- | --- |
|1|1000|2.6034818|26.298761|
|1|5000|5.9483757|35.55671|
|1|10000|9.74156|44.326374|
|1|50000|34.43163|105.99434|
|1|100000|67.15671|798.17957|
|10|100|4.862188|28.975754|
|10|500|7.8796077|33.293507|
|10|1000|10.552007|44.25754|
|10|5000|42.58707|313.13174|
|10|10000|74.20953|834.67|

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Average (%) | Memory Average (MB) |
| --- | --- | --- | --- |
|1|1000|2.1035452|26.90423|
|1|5000|5.6897674|31.05105|
|1|10000|9.672566|42.60291|
|1|50000|35.776554|125.39278|
|1|100000|64.104744|190.23087|
|10|100|3.7759998|29.441944|
|10|500|7.741545|33.589844|
|10|1000|14.586588|79.55846|
|10|5000|42.9494|119.6844|
|10|10000|71.79728|675.9653|

File: scripts/unix-install.sh [hunk 1]
```diff
@@ -405,23 +408,26 @@ install_package()
...
+  curl -L "$agent_download_url" -o "$agent_binary" --progress-bar --fail || error_exit "$LINENO" "Failed to download package"
   succeeded
 
   info "Setting permissions..."
   chmod +x "$agent_binary"
   ln -sf "$agent_binary" "/usr/local/bin/$BINARY_NAME"
   succeeded
 
+  info "Downloading plugins..."
+  curl -L "$plugins_download_url" -o plugins.tar.gz --progress-bar --fail || error_exit "$LINENO" "Failed to download plugins"
```
Comment (camdencheek) [inline scripts/unix-install.sh:425]: We should probably download this to a temp directory, and remove it when we're done with it

Comment (camdencheek) [reply scripts/unix-install.sh:425]: You could probably also just pipe directly to stdin of `tar`

**Comment by djaglowski** [comment]
image

Validated that plugins are being fetched and stored with release assets.

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Average (%) | Memory Average (MB) |
| --- | --- | --- | --- |
|1|1000|2.7069907|28.521955|
|1|5000|6.0346913|36.302532|
|1|10000|9.672617|42.26293|
|1|50000|34.82844|126.61018|
|1|100000|65.60466|200.68723|
|10|100|4.4656177|27.340652|
|10|500|8.017439|35.469288|
|10|1000|10.913994|43.796604|
|10|5000|41.91476|93.5334|
|10|10000|82.10615|197.94208|

**Pull request merged**